benmarks.py

```
@@ -0,0 +1,842 @@
+                treasury['statusType'] == 'Active' and
+                (treasury['tier'] != 'Sec' or treasury['tier'] == 'Sr Sec') and
+                treasury['organizationDetails']['issuerName'] == 'United States of America' and
+                treasury['bd_currentCoupon'] is not None and
+                treasury['bd_maturityDate'] is not None and
+                treasury['cusip'] is not None and 
+                treasury['isin'] is not None and
+                treasury['currentAmountOutstanding'] is not None and
+                maturityDate > today):
+            return True
+        else:
+            return False
```

Ruff (0.8.2)

Return the condition directly

Inline condition

(SIM103)